circle.yml

```diff
@@ -0,0 +1,25 @@
+dependencies:
+    cache_directories:
+       - "~/.opam"
+       - "~/aptcache"
+    override:
+        - sudo add-apt-repository -y ppa:avsm/ppa
+        - sudo mkdir -p ~/aptcache/partial
+        - sudo touch ~/aptcache/lock
```

Comment:
Oh I had to add this one time. Guess I could have done that via SSH. Can remove.